### PR TITLE
Publish Cache job Inputs

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

File: engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala [hunk 1]
```diff
@@ -317,10 +319,10 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
   }
 
   when(WorkflowExecutionInProgressState) {
-    case Event(JobRunning(jobDescriptor, backendJobExecutionActor), stateData) =>
+    case Event(JobRunning(jobDescriptor, Some(backendJobExecutionActor)), stateData) =>
       pushRunningJobMetadata(jobDescriptor)
       stay() using stateData
-        .addBackendJobExecutionActor(jobDescriptor.key, backendJobExecutionActor)
+        .addBackendJobExecutionActor(jobDescriptor.key, Some(backendJobExecutionActor))
```
Comment (cjllanwarne) [inline engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala]: Personally I prefer `Some` but the prevailing tyranny is `Option`

File: engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala [hunk 2]
```diff
@@ -317,10 +319,10 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
   }
 
   when(WorkflowExecutionInProgressState) {
-    case Event(JobRunning(jobDescriptor, backendJobExecutionActor), stateData) =>
+    case Event(JobRunning(jobDescriptor, Some(backendJobExecutionActor)), stateData) =>
```
Comment (cjllanwarne) [inline engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala]: Should catch both cases and it's the updated data which depends on Some vs None

**Comment by cjllanwarne** [comment]
👍

**Comment by cjllanwarne** [comment]
If these changes are working, could you merge direct to develop?

**Comment by ruchim** [comment]
@cjllanwarn, I could do like a cherry pick onto develop?

File: engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala [hunk 3]
```diff
@@ -30,6 +30,8 @@ import wdl4s.util.TryUtil
 import wdl4s.values.{WdlArray, WdlValue}
 import wdl4s.{Scope, _}
 
+import scala.Option
+import scala.Option._
```
Comment (mcovarr) [inline engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala]: looks like a bit too much importing enthusiasm

**Comment by mcovarr** [comment]
:+1:

File: engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala (hunk 3 above)
Comment (ruchim) [inline engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala]: I couldn't import Option, it was driving me crazy

Comment (cjllanwarne) [inline engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala]: Now that the Some's are gone, you might be able to remove these

**Pull request merged**